editor/plugins/material_editor_plugin.cpp

```diff
@@ -102,6 +122,13 @@ void MaterialEditor::edit(Ref<Material> p_material, const Ref<Environment> &p_en
 	} else {
 		hide();
 	}
+
+	rot.x = Math::deg_to_rad(-15.0);
+	rot.y = Math::deg_to_rad(30.0);
+	_update_rotation();
+
+	box_instance->set_transform(Transform3D() * 0.25);
+	sphere_instance->set_transform(Transform3D() * 0.375);
```

Comment:
Move these two lines to where `Transform3D box_xform;` is in the constructor, and remove box_xform. These only need to be set once, so we should put them in the constructor, and they make all the box_xform code irrelevant since these lines override it.

Reply:
Done.